job/server/src/main/java/alluxio/job/plan/migrate/MigrateDefinition.java

```diff
@@ -339,6 +339,14 @@ private static void migrate(MigrateCommand command, WritePType writeType, boolea
     String destination = command.getDestination();
     LOG.debug("Migrating {} to {}", source, destination);
 
+    // If the write type is async through and the source is being deleted but source is persisted,
+    // keep the destination to be persisted by changing its write type
+    if (deleteSource && writeType.equals(WritePType.ASYNC_THROUGH)) {
+      if (fileSystem.getStatus(new AlluxioURI(source)).isPersisted()) {
+        writeType = WritePType.THROUGH;
```

Comment:
This means it will NOT be CACHE. Is that what we want?

Reply:
I don't actually know. Do you prefer CACHE_THROUGH here?

Reply:
Ping.